etas/evaluation.py

```diff
@@ -0,0 +1,438 @@
+import sys
+from etas.inversion import ETASParameterCalculation, read_shape_coords, polygon_surface, round_half_up, parameter_dict2array, haversine
+from etas.simulation import simulate_aftershock_time
+import pandas as pd
+import geopandas as gpd
+from shapely.geometry import Polygon
+from joblib import Parallel, delayed
+from tabulate import tabulate
+import numpy as np
+from scipy.special import expi
+import json
+from scipy.integrate  import quad
+from shapely.geometry import Polygon, Point, LineString
+import matplotlib.pyplot as plt
+
+
+def compute_dist_squared_from_i(i,lat_rads: np.ndarray,long_rads: np.ndarray,earth_radius=6.3781e3):
+
+    return np.square(
+                haversine(
+                    lat_rads[i],
+                    lat_rads[:i],
+                    long_rads[i],
+                    long_rads[:i],
+                    earth_radius,
+                )
+            )
+
+def to_days(timediff):
```

Comment:
could probably import the function from inversion.py? or not?

Reply:
You're right, although this one in evaluation.py uses numpy timedelta64, whereas the one in inversion.py uses pandas dt.timedelta.

The repeat definition should be avoided nonetheless.